reference/conanfile/attributes.rst

```diff
@@ -230,7 +230,15 @@ There is an special value ``ANY`` to allow any value for a given option. The ran
     class MyPkg(ConanFile):
         ...
         options = {"shared": [True, False], "commit": "ANY"}
+        default_options = "shared=False", "commit=None"
 
+        def build(self):
+            if not self.options.commit:
+                self.output.info("This evaluates to True")
+            if self.options.commit == "None":
```

Comment:
Please explain clearly that this is not recommended. Only the first way is recommended.